server/server.go

```diff
@@ -540,10 +541,9 @@ func (s *Server) startServerLoop(ctx context.Context) {
 	go s.etcdLeaderLoop()
 	go s.serverMetricsLoop()
 	go s.encryptionKeyManagerLoop()
-	if s.IsAPIServiceMode() { // disable tso service and resource manager service in api server
-		s.serverLoopWg.Add(2)
+	if s.IsAPIServiceMode() { // disable tso service in api server
+		s.serverLoopWg.Add(1)
 		go s.watchServicePrimaryAddrLoop(mcs.TSOServiceName)
-		go s.watchServicePrimaryAddrLoop(mcs.ResourceManagerServiceName)
```

Comment:
Why remove it?

Reply:
If remove it, the related test also need to be deleted or commented.

Reply:
We don't need it for now.